VagrantProvision.sh

```diff
@@ -358,159 +358,8 @@ if [ -f $HOOT_HOME/hoot-services/src/main/resources/conf/local.conf ]; then
     rm -f $HOOT_HOME/hoot-services/src/main/resources/conf/local.conf
 fi
 
-cd ~
```

Comment:
She we change this in the Ubuntu16.04 provisioner as a courtesy?  Or are they not installing hadoop on that platform?

Reply:
I didn't touch the 16.04 script because it said: [NOTE: Hadoop install has been removed](https://github.com/ngageoint/hootenanny/blob/develop/VagrantProvisionUbuntu1604.sh#L422-L424).

Reply:
:+1: